base/compiler/tfuncs.jl

```diff
@@ -2554,6 +2560,32 @@ function memoryop_noub(@nospecialize(f), argtypes::Vector{Any})
     return false
 end
 
+function current_scope_tfunc(interp::AbstractInterpreter, sv::InferenceState)
+    pc = sv.currpc
+    while true
+        handleridx = sv.handler_at[pc][2]
```

Comment:
```suggestion
        handleridx = sv.handler_at[pc][1]
```
Shouldn't this look at the active `try` region rather than the `catch` region?

Reply:
Good catch